lua/lean/util.lua

```diff
@@ -0,0 +1,138 @@
+local uv = vim.loop
+local M = {}
+
+-- M.path copied from lspconfig/util to avoid creating a dependency...
```

Comment:
lspconfig is Apache 2.0 licensed so in theory we're supposed to add the little blurb here.